pkg/render/helm.go

```diff
@@ -32,5 +32,12 @@ func ToChart(name string, m *manifest.Manifest, options fleet.BundleDeploymentOp
 		return nil, err
 	}
 
-	return m.ToTarGZ()
+	var decrypt bool
+	if *options.Decrypt == nil || *options.Decrypt {
+		decrypt = true
+	} else {
+		decrypt = false
+	}
```

Comment:
For pointer safety, I would prefer that we set it to false by default.
```go
decrypt := false
if *options.Decrypt == nil || *options.Decrypt {
    decrypt = true
}
```

Reply:
github is having some issues today, hopefully the change will show up soon (already pushed on my branch)